src/axom/core/CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ blt_list_append( TO core_depends ELEMENTS caliper IF CALIPER_FOUND )
 blt_list_append( TO core_depends ELEMENTS camp IF CAMP_FOUND )
 blt_list_append( TO core_depends ELEMENTS umpire IF UMPIRE_FOUND )
 blt_list_append( TO core_depends ELEMENTS RAJA IF RAJA_FOUND )
+blt_list_append( TO core_depends ELEMENTS conduit::conduit IF CONDUIT_FOUND )
```

Comment:
Question for the team: is everyone OK with this change? It makes all Axom components optionally-dependent on Conduit.

Reply:
Background: This is to set a variable to the `conduit` type matching our `IndexType` so we don't have to check every time we need it.    It's for convenience but not essential.
https://github.com/LLNL/axom/blob/d4a5db831196801cd46f05418c0791de726fc906/src/axom/core/Types.hpp#L67-L79

Reply:
@gunney1 understood. I want others to weigh in on this change.

Reply:
I needed something like that in my MIR PR but I made [type traits called cpp2conduit](https://github.com/LLNL/axom/blob/feature/whitlock/mir_update/src/axom/mir/utilities/blueprint_utilities.hpp) that let me get that id. In past discussions with Kenny, we think ultimately the blueprint support from MIR, and probably quest, may need to be separated out into a new shared Axom component (unless that right place is core). Until that happens, maybe these TypeIDs live in a quest header.

Reply:
I like the @BradWhitlock solution to this.

Reply:
I like @BradWhitlock's solution also, but doesn't it also make all of axom dependent on Conduit when we configure with Conduit?

Reply:
I'll move the code into quest to avoid changing core,  Will use @BradWhitlock's solution when it is available.